IDL/cocoplot.pro

```diff
@@ -16,30 +16,54 @@ function COCOPLOT, coco_datacube, filter, rgbthresh=rgbthresh, threshmethod=thre
 ;	Result = COCOPLOT(coco_datacube, filter)
 ;
 ; INPUTS:
-;	  coco_datacube:  Input data cube of dimensions [nx, ny, nspect_points]
+;   coco_datacube:  Input data cube of dimensions [nx, ny, nspect_points]
 ;   filter:         Filter of dimensions [nspect_points, 3] 
 ;
 ; KEYWORD PARAMETERS:
-;	  rgbthresh:      Flag to apply saturation thresholding. Defaults to not set.
+;   rgbthresh:      Flag to apply saturation thresholding. Defaults to not set.
 ;   threshmethod:   Scalar string specifying the Saturation thresholding method.
 ;                   Can be 'fraction', 'numeric' or 'percentile'. Defaults to not set.
-;   Note: keyword settings are passed on to COCORGB() and COCONORM(), and do not
-;   have an effect inside COCOPLOT.
+;   rgb_in:         Include if you are handing an already color collapsed RGB

```

Comment:
Probably simpler to check for the presence of `filter`, rather than adding a keyword `rgb_in`. If `filter` is absent, `cocoplot` should assume `coco_datacube` to be `[nx,ny,3]` and `[nx,ny,nspect_points]` otherwise. Should still check for `sz[0] eq 3` and `sz[3] eq 3` (as it already does) if no filter is supplied, just to be sure.

Reply:
Filter now a keyword and replaces RGB_IN, see latest commit.